docs/api/overview.md

```diff
@@ -7,6 +7,6 @@ expanded: Client APIs
 
 There are various client APIs for DuckDB. DuckDB's "native" API is [C++](cpp), with "official" wrappers available for [C](c/overview), [Python](python/overview), [R](r), [Java](java), [Node.js](nodejs/overview), [WebAssembly/Wasm](wasm/overview), [ODBC API](odbc/overview), [Julia](julia), and a [Command Line Interface (CLI)](cli).
 
-There are also contributed third-party DuckDB wrappers for [Ruby](https://github.com/suketa/ruby-duckdb), [Go](https://github.com/marcboeker/go-duckdb), [C#](https://github.com/Giorgi/DuckDB.NET), [Rust](https://github.com/wangfenjin/duckdb-rs) and [Common Lisp](https://github.com/ak-coram/cl-duckdb).
+There are also contributed third-party DuckDB wrappers for [Ruby](https://github.com/suketa/ruby-duckdb), [Go](https://github.com/marcboeker/go-duckdb), [C#](https://github.com/Giorgi/DuckDB.NET), [Rust](https://github.com/wangfenjin/duckdb-rs), [Common Lisp](https://github.com/ak-coram/cl-duckdb), and [Crystal](https://github.com/amauryt/crystal-duckdb).
```

Comment:
This list is getting kinda long - maybe make it a markdown list and add author names?

Reply:
Done. Changed it to a list in alphabetical order and also added the respective authors.